src/partials/languages/id.html

```diff
@@ -25,19 +25,37 @@
   "language": "id",
   "action.edit": "Ubah halaman ini",
   "action.skip": "Lewati ke isi",
-  "clipboard.copy": "Salin ke memori",
-  "clipboard.copied": "Tersalin ke memori",
+  "action.view": "Lihat sumber halaman ini",
+  "announce.dismiss": "Jangan lihat ini lagi",
+  "clipboard.copy": "Salin ke clipboard",
+  "clipboard.copied": "Tersalin ke clipboard",
+  "consent.accept": "Terima",
+  "consent.manage": "Kelola pengaturan",
+  "consent.reject": "Tolak",
   "footer.next": "Selanjutnya",
   "footer.previous": "Sebelumnya",
   "meta.comments": "Komentar",
   "meta.source": "Sumber",
+  "nav": "Navigasi",
+  "search": "Cari",
   "search.config.pipeline": " ",
   "search.placeholder": "Cari",
+  "search.share": "Bagikan",
+  "search.reset": "Kosongkan",
+  "search.result.initializer": "Initializing search",
   "search.result.placeholder": "Ketik untuk mulai pencarian",
   "search.result.none": "Tidak ada dokumen yang sesuai",
   "search.result.one": "1 dokumen ditemukan",
   "search.result.other": "# dokumen ditemukan",
-  "source": "Menuju repositori",
+  "search.result.more.one": "1 lagi di halaman ini",
+  "search.result.more.other": "# lagi di halaman ini",
+  "search.result.term.missing": "Tidak ada",
+  "select.language": "Pilih bahasa",
+  "select.version": "Pilih versi",
+  "source": "Ke repositori",
   "source.file.date.created": "Dibuat",
+  "source.file.date.updated": "Pembaruan terakhir",
+  "tabs": "Tab",
   "toc": "Daftar isi"
```

Comment:
A comma is missing here – the build will not work.

Reply:
This has been fixed.